website/src/_posts/2022-09-3.0.md

```diff
@@ -0,0 +1,146 @@
+
+<img class="border" alt="Uppy Compressor plugin showing compressed images notification" src="/images/blog/2.4-2.7/compressor.jpg">
+
+## Remote Sources
+
+We’ve introduced a new Uppy plugin/preset that combines Instagram, Facebook, Google Drive, Box, Unsplash, Dropbox, One Drive, Zoom and any future remote cloud sources that Uppy will support. It only works with Dashboard and let’s you enable all the above sources in only line:
+
+```js
+uppy.use(RemoteSoruces, { companionUrl: 'https://companion.uppy.io' }) // Remote Soruces need Companion server instance to work, you can get one hosted from Transloadit.com or self-host it
+```
+
+## Robodog is Deprecated
+
+Uppy is flexible and extensible through plugins. But the integration code could sometimes be daunting. This is what brought [Robodog](/docs/robodog/) to life. An alternative with the same features, but with a more ergonomic and minimal API.
+
+But, it didn’t come without its own set of new problems:
+
+* It tries to do the exact same, but it looks like a different product.
+* It’s confusing for users whether they want to use Robodog or Uppy directly.
+* Robodog is more ergonomic because it’s limited. When you hit such a limit, you
+  need to refactor everything to Uppy with plugins.
+
+This has now led us to deprecating Robodog and embrace Uppy for its strong suits: modularity and flexibility. At the same time, we also introduced something to take away some repetitive integration code: [`@uppy/remote-sources`](/docs/remote-sources).
+
+To mimic the Robodog implementation with all its features, you can use the code snippet below. But chances are Robodog did more than you need so feel free to remove things or go through the [list of plugins](/docs/plugins/) and install and use the ones you need.
+
+You can also checkout how we migrated the Robodog example ourselves in this [commit](https://github.com/transloadit/uppy/commit/089aaed615c77bafaf905e291b6b4e82aaeb2f6f).
+
+```js
+// Import just the plugins you need
+import Uppy from '@uppy/core'
+import Dashboard from '@uppy/dashboard'
+import RemoteSources from '@uppy/remote-sources'
+import Webcam from '@uppy/webcam'
+import ScreenCapture from '@uppy/screen-capture'
+import ImageEditor from '@uppy/image-editor'
+import Audio from '@uppy/audio'
+import Transloadit, { COMPANION_URL, COMPANION_ALLOWED_HOSTS } from '@uppy/transloadit'
+
+// Import their styles
+import '@uppy/core/dist/style.css'
+import '@uppy/dashboard/dist/style.css'
+import '@uppy/webcam/dist/style.css'
+import '@uppy/audio/dist/style.css'
+import '@uppy/screen-capture/dist/style.css'
+import '@uppy/image-editor/dist/style.css'
+
+// Initialize Uppy
```

Comment:
nit: these code comments are too obvious to add any value. It's like have a comment `// variable` above `const foo = 'bar'`.

```suggestion
import Uppy from '@uppy/core'
import Dashboard from '@uppy/dashboard'
import RemoteSources from '@uppy/remote-sources'
import Webcam from '@uppy/webcam'
import ScreenCapture from '@uppy/screen-capture'
import ImageEditor from '@uppy/image-editor'
import Audio from '@uppy/audio'
import Transloadit, { COMPANION_URL, COMPANION_ALLOWED_HOSTS } from '@uppy/transloadit'

import '@uppy/core/dist/style.css'
import '@uppy/dashboard/dist/style.css'
import '@uppy/webcam/dist/style.css'
import '@uppy/audio/dist/style.css'
import '@uppy/screen-capture/dist/style.css'
import '@uppy/image-editor/dist/style.css'

```